sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -0,0 +1,853 @@
+        """
+        entry_ele = await self._get_entity_element(queue_name, **kwargs)
+        entry = QueueDescriptionEntry.deserialize(entry_ele)
+        if not entry.content:
+            raise ResourceNotFoundError("Queue '{}' does not exist".format(queue_name))
+        queue_description = QueueDescription._from_internal_entity(
+            entry.content.queue_description)
+        queue_description.name = queue_name
+        return queue_description
+
+    async def get_queue_runtime_info(self, queue_name: str, **kwargs) -> QueueRuntimeInfo:
+        """Get the runtime information of a queue.
+
+        :param str queue_name: The name of the queue.
+        :rtype: ~azure.servicebus.management.QueueRuntimeInfo
+        """
+        entry_ele = await self._get_entity_element(queue_name, **kwargs)
+        entry = QueueDescriptionEntry.deserialize(entry_ele)
+        if not entry.content:
+            raise ResourceNotFoundError("Queue {} does not exist".format(queue_name))
+        runtime_info = QueueRuntimeInfo._from_internal_entity(
+            entry.content.queue_description)
+        runtime_info.name = queue_name
+        return runtime_info
+
+    async def create_queue(self, queue: Union[str, QueueDescription], **kwargs) -> QueueDescription:
+        """Create a queue.
+
+        :param queue: The queue name or a `QueueDescription` instance. When it's a str, it will be the name
+         of the created queue. Other properties of the created queue will have default values decided by the
+         ServiceBus. Use a `QueueDescription` if you want to set queue properties other than the queue name.
+        :type queue: Union[str, ~azure.servicebus.management.QueueDescription]
+        :rtype: ~azure.servicebus.management.QueueDescription
+        """
+        try:
+            queue_name = queue.name  # type: ignore
+            to_create = queue._to_internal_entity()  # type: ignore
+        except AttributeError:
+            queue_name = queue  # type: ignore
+            to_create = InternalQueueDescription()  # Use an empty queue description.
+
+        create_entity_body = CreateQueueBody(
+            content=CreateQueueBodyContent(
+                queue_description=to_create,  # type: ignore
+            )
+        )
+        request_body = create_entity_body.serialize(is_xml=True)
+        with _handle_response_error():
+            entry_ele = cast(
+                ElementTree,
+                await self._impl.entity.put(
+                    queue_name,  # type: ignore
+                    request_body, api_version=constants.API_VERSION, **kwargs)
+            )
+
+        entry = QueueDescriptionEntry.deserialize(entry_ele)
+        result = QueueDescription._from_internal_entity(
+            entry.content.queue_description)
+        result.name = queue_name
+        return result
+
+    async def update_queue(self, queue_description: QueueDescription, **kwargs) -> None:
```

Comment:
Since this is AIO, can't we do proper kwonly args?  (e.g. foo, bar, *, kwfoo=None, kwbar=None)

Reply:
Yes, we definitely should here :)

Reply:
Changed to use named params